index.php

```diff
@@ -41,9 +41,8 @@ function page_snippetslist_tab($page)
 
 function grouped_snippets()
 {
-    $snippets =Record::find(array(
+    $snippets =Snippet::find(array(
```

Comment:
Did the API change for this in 0.8.0 ?
If so, version bump should be done, for Easysnippet and required Wolf version.